bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```diff
@@ -11,7 +11,13 @@ export default class Whiteboard extends React.Component {
   }
 
   renderWhiteboard() {
+    let slideObj = this.props.current_slide;
```

Comment:
Sure, I will change this in the next PR since I committed all the fixes already.